### PR TITLE
Add table of contents to CONTRIBUTING

### PR DESCRIPTION
**Pull request by lorensr**
And minor clarity/typo fixes

### CODE REVIEW

**Review by bergundy — APPROVED** [review]
Thanks!

We have to fix the fact that when you recompile typescript from scratch it changes the permissions on `packages/create-project/lib/index.js`, it might be possible to not commit it in the first place (but then it wouldn't be runnable from a github URL).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,14 +1,30 @@
 # How to Contribute
 
-The Node.js SDK, as well as the rest of the Temporal codebase is open sourced under the MIT license.
+The Node.js SDK (as well as the rest of the Temporal codebase) is open sourced under the MIT license.
 
-We welcome contributions from the community. To contribute please start by opening an [issue](https://github.com/temporalio/sdk-node/issues) and dicussing the proposed change, once a change has been agreed upon development may start and submitted via a [pull request](https://github.com/temporalio/sdk-node/pulls).
+We welcome contributions from the community. To contribute please start by opening an [issue](https://github.com/temporalio/sdk-node/issues) and discussing the proposed change. Once a change has been agreed upon, development may start and be submitted via a [pull request](https://github.com/temporalio/sdk-node/pulls).
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```
Comment (bergundy) [inline CONTRIBUTING.md:7]: Maybe add a little explanation or an npm script to generate this TOC?

Comment (lorensr) [reply CONTRIBUTING.md:9]: Added a script and also applied it to README

File: packages/meta/README.md [hunk 2]
```diff
@@ -12,6 +12,19 @@ Temporal is a microservice orchestration platform which enables developers to bu
 
 Temporal is a mature technology—it's a fork of Uber's Cadence. Temporal is being developed by [Temporal Technologies](https://temporal.io), a startup by the creators of Cadence.
 
+**Contents**:
```
Comment (bergundy) [inline packages/meta/README.md:15]: I don't feel like this is needed for such a short file, also GH auto generates TOCs for markdown.

**Comment by bergundy** [comment]
@lorensr up to you if you want to add the TOC or not, I don't have a strong opinion here.
The PR is approved so feel free to merge whenever CI passes.

**Comment by bergundy** [comment]
Ended up manually pushing this to main.
I didn't take the TOC, just the docs and typo fixes.
Thanks.

**Pull request closed without merging**